diff --git a/boost/boost.bzl b/boost/boost.bzl
index 6d6d488..a0776ea 100644
--- a/boost/boost.bzl
+++ b/boost/boost.bzl
@@ -224,6 +224,7 @@ def boost_deps():
             name = "boost",
             build_file = "@com_github_nelhage_rules_boost//:BUILD.boost",
             patch_cmds = ["rm -f doc/pdf/BUILD"],
+            patch_cmds_win = ["Remove-Item -Force doc/pdf/BUILD"],
             sha256 = "d73a8da01e8bf8c7eda40b4c84915071a8c8a0df4a6734537ddde4a8580524ee",
             strip_prefix = "boost_1_71_0",
             urls = [
